gitapp/lib/users_list/users_list.dart

```diff
@@ -0,0 +1,46 @@
+import 'package:flutter/material.dart';
+import 'package:gitapp/users_list/user.dart';
+import "package:http/http.dart" as http;
+import 'dart:convert' as convert;
+
+class UsersList {
+  List<User> userlist = List.empty(growable: true);
+  List<User> searchedlist = List.empty(growable: true);
```

Comment:
Please use camel case in all examples so 

searchedList

Reply:
solved